.github/workflows/action-image-scan-deploy-dashing.yaml

```diff
@@ -0,0 +1,31 @@
+on: 
+  push:
+    branches:
+      - master
+      - main
+
+env:
+  ROS_VERSION: 2
+  UBUNTU_DISTRO: bionic
+  ROS_DISTRO: foxy
+
+jobs:
+  build-secure-and-push:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+
+    - run: docker build --build-arg ROS_VERSION=$ROS_VERSION --build-arg UBUNTU_DISTRO=$UBUNTU_DISTRO --build-arg ROS_DISTRO=$ROS_DISTRO --target devel-build -t ros$ROS_VERSION-$ROS_DISTRO:devel .
+      
+    - uses: Azure/container-scan@v0
```

Comment:
What does this scan look at?

Reply:
see [here](https://github.com/goodwithtech/dockle/blob/master/CHECKPOINT.md)